### ISSUE TITLE
Playwright does not run anymore in CI on Code OSS

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#160875]
https://dev.azure.com/vscode/VSCode/_build/results?buildId=73459&view=logs&j=a8a6689c-5a84-59c0-482a-7dd9f21eb2d8&t=1fa6fb8d-7bb3-5acd-5778-bf99aac7f637

**Comment by bpasero** [comment]
🤷

**Comment by lszomoru** [comment]
Pushed [a fix](https://github.com/microsoft/vscode/pull/160878) to "pin" the image that we are using in order to unblock the pipeline. But we still have to understand what has changed between versions and we need to see how we should adapt to those changes. One thing to keep in mind is that we did try to use `npx playwright install-deps` but that did not resolve the problem so we might have to engage with the Playwright team.
